.vuepress/config.js

```diff
@@ -138,12 +138,27 @@ module.exports = {
           path: '/create/how_to/',
           collapsable: false,
           children: [
-            '/create/how_to/running_production',
-            '/create/how_to/search_bar_for_docs',
-            '/create/how_to/digitalocean_droplet',
+            {
+              title: 'Run MeiliSearch in Production',
+              path: '/create/how_to/running_production',
+            },
+            {
+              title: 'Integrate MeiliSearch into Your Docs',
+              path: '/create/how_to/search_bar_for_docs',
+            },
+            {
+              title: 'Deploy MeiliSearch on DigitalOcean',
+              path: '/create/how_to/digitalocean_droplet',
+            },
```

Comment:
```suggestion
            {
              title: 'Run in Production',
              path: '/create/how_to/running_production',
            },
            {
              title: 'Integrate into Your Docs',
              path: '/create/how_to/search_bar_for_docs',
            },
            {
              title: 'Deploy on DigitalOcean',
              path: '/create/how_to/digitalocean_droplet',
            },
```

Since we are on the docs of meilisearch, we could remove its name from the title to make titles shorter? What do you think

Reply:
Totally agree!